File: app/src/main/java/ru/gsench/githubusers/presentation/view/UserListView.java
package ru.gsench.githubusers.presentation.view;

/**
 * Created by grish on 01.04.2017.
 */

public interface UserListView {

    public void init();
    public void notifyUsersAdded(int offset, int count);
    public void clearList();
    public void showLoadingError();
    public void showParseError();
    public void showUnexpectedError();
    public void closeView();
}

File: app/src/main/java/ru/gsench/githubusers/presentation/view/UserListView.kt
package ru.gsench.githubusers.presentation.view

/**
 * Created by grish on 01.04.2017.
 */

interface UserListView {

    fun init()
    fun notifyUsersAdded(offset: Int, count: Int)
    fun clearList()
    fun showLoadingError()
    fun showParseError()
    fun showUnexpectedError()
    fun closeView()
    fun hideLoading()
    fun showLoading()
    fun notifyUserChanged(i: Int)
}
